From 914a996e12bbca161fc0ef7a0c07ef1c857ffde6 Mon Sep 17 00:00:00 2001
From: Noah Pendleton <2538614+noahp@users.noreply.github.com>
Date: Mon, 6 Jun 2022 16:46:37 -0400
Subject: [PATCH] Workaround building on Mac

This file casuses a lot of sadness when building on mac. Remove it.
---
 intl/VERSION | 1 -
 1 file changed, 1 deletion(-)
 delete mode 100644 intl/VERSION

diff --git a/intl/VERSION b/intl/VERSION
deleted file mode 100644
index 13031835..00000000
--- a/intl/VERSION
+++ /dev/null
@@ -1 +0,0 @@
-GNU gettext library from gettext-0.12.1
-- 
2.36.1

